printer_mods/khay/V2_3x_Keystone_Plugpanel/README.md

```diff
@@ -0,0 +1,10 @@
+Adding an unnecessary amount of Keystone Holders to the Plug_Panel component. 
+The part works with "TycoElectronics - 10EGG1-2 (Normal) or 10EHG1-2 (Medical) Filtered Power Inlet" or equal Inlets and needs moving the switch to another skirt 
+like with @tayto-chip´s mod.
+
+![Plug_Panel](Img/Plug_Panel_CAD.png)
+
+![Plug_Panel](Img/DSC_3253.JPG)
+
+tayto-chip´s "skirt_switch_mod":
```

Comment:
Can you use a relative link?  So these two lines become

    tayto-chip's [Skirt Switch Mod](../../tayto-chip/skirt_switch_mod)

Reply:
Hello KiloQubit, updated filename and README for plug_panel mod. 

That chamfer on the 4020 Skirt is the same as in the original V2.4 Skirts, I left that untouched to save me some trouble remodeling. If you want me to change it, pls just comment and I will give that some more rounding off.

Reply:
I didn't realize the stock skirts were like that, this is fine then.